.travis.yml

```diff
@@ -57,25 +44,21 @@ matrix:
     - php: 7.1
       env:
         - DEPS=latest
-    - php: hhvm 
+    - php: 7.2
       env:
         - DEPS=lowest
-    - php: hhvm 
+    - php: 7.2
       env:
         - DEPS=locked
-    - php: hhvm 
+    - php: 7.2
       env:
         - DEPS=latest
   allow_failures:
-    - php: hhvm
-
-notifications:
-  irc: "irc.freenode.org#zftalk.dev"
-  email: false
+    - php: 7.2
```

Comment:
Now shouldn't be allowed to fail, right? :)

Reply:
yes, its a pr from august 2017
i dont want to update it each month and wait more and more